.github/workflows/Test.yml

```diff
@@ -15,12 +15,30 @@ jobs:
   test:
     name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
     runs-on: ${{ matrix.os }}
+    timeout-minutes: 180
+    permissions: # needed to allow julia-actions/cache to proactively delete old caches that it has created
+      actions: write
+      contents: read
     strategy:
       fail-fast: false
       matrix:
-        version: ['1.10', '1.11']
-        os: ['ubuntu-latest', 'macOS-latest']
-        arch: [x64]
+        version:
+          - "1"
+          - lts
```

Comment:
Why this change? GPU back-ends typically depend much more strongly on the specific Julia version, so I think it's better to keep it like it was.

Reply:
Fixed